### DISCUSSION TITLE
Auto Labeling

### DISCUSSION THREAD

**Discussion by ggoodd-any** [wkentaro/labelme#1261] [Ideas / Feature requests]
Yolov5 7.0 integrates classification, detection, and instance segmentation functions. Perhaps we can integrate a Yolov5 7.0 inference module to make the labelme have semi-automatic annotation functionality. Furthermore, we can integrate the training module of YOLOv5 to achieve a semi-automatic annotation loop for human label -->train model -->model predict -->human label.

**Comment by vietanhdev** [comment]
Please check out [LabelAnything](https://github.com/vietanhdev/anylabeling) - A fork of labelme with Segment Anything and YOLO models for auto labeling. 

Training loop instructions will be added soon.
![image](https://user-images.githubusercontent.com/18329471/232065045-683a61de-e363-4343-8946-addcdd529eb6.png)

Demo video: https://www.youtube.com/watch?v=5iQSGL7ebXE
